pkg/kapp/cmd/core/config_factory.go

```diff
@@ -69,6 +78,11 @@ func (f *ConfigFactoryImpl) RESTConfig() (*rest.Config, error) {
 		return nil, fmt.Errorf("Building Kubernetes config%s: %s%s", prefixMsg, err, hintMsg)
 	}
 
+	if f.qps > 0.0 {
```

Comment:
So, if `qps` is not set, we ignore burst config. Is that intentional?

Reply:
intentional as a form of sanity check. if qps is 0.0, ConfigureClient wasnt probably called and we just dealing with zero values from the struct -- in which case we dont want to set them to 0s.